src/Components/Bounty/style.css

```diff
@@ -0,0 +1,17 @@
+.bountyContainer {
+    background-color: pink
+}
+
+.bountyTitle {
+    background-color: lightblue
+}
+
+.bountyInfo {
+    background-color: green
+}
+
+
+.bountyReward{
+    background-color: gold
```

Comment:
Use color hex and pull these colors out to css variables. You can pick your colors with a google chrome extension called color picker or w/e. Have a main.css and import all your other styles into your main.css you can put these color variables into the variables.css file and then import that into your main.css and in your index.html all you do is link up the main.css contact me for more clarification